### PR TITLE
Reformatted the csv files and plotting notebooks

### PR DESCRIPTION
**Pull request by yuema137**
Thanks Knut for this helpful tool! I noticed that the files are in different format so I reformatted all of them into csv. Also I added a jupyter notebook to:
-  search for existing limit by name
-  make quick plots based on csv files

An example plot is attached below:
image

### CODE REVIEW

File: data/CRESST2_SI_2016.csv
Comment (kdund) [inline data/CRESST2_SI_2016.csv:1]: I think as a general matter the .csv parsers can deal with extra spaces? Better then to let them than to impose a very strict standard?

File: data/PandaX-4T_2107.13438_SI_WIMP.csv
Comment (kdund) [inline data/PandaX-4T_2107.13438_SI_WIMP.csv:1]: The header here was distorted

File: data/xenon1t_1ty.csv
Comment (kdund) [inline data/xenon1t_1ty.csv:1]: Also here the header was distorted by the conversion

**Review by kdund — CHANGES_REQUESTED** [review]
Hi, @yuema135, thanks a lot! 
I propose you go through and fix the header and then we can merge-- after that,I plan on doing some filename renaming and start thinking about the separate-metadata-yaml file :)

File: data/xenon1t_1ty.csv [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-30,4.05E-47,1.07E-47,1.84E-47,3.85E-47,6.80E-47,9.51E-47  
-40,4.48E-47,9.59E-48,1.51E-47,3.39E-47,6.07E-47,9.54E-47  
-50,5.26E-47,1.03E-47,1.61E-47,3.58E-47,6.06E-47,9.37E-47  
-70,6.79E-47,9.82E-48,1.75E-47,4.06E-47,7.09E-47,1.15E-46  
-100,9.12E-47,1.25E-47,2.37E-47,4.97E-47,8.97E-47,1.34E-46  
-150,1.31E-46,1.66E-47,3.34E-47,6.87E-47,1.25E-46,1.83E-46  
-200,1.72E-46,2.48E-47,4.31E-47,9.04E-47,1.50E-46,2.24E-46  
-400,3.34E-46,4.02E-47,8.09E-47,1.69E-46,3.02E-46,4.52E-46  
-1000,8.24E-46,1.00E-46,1.79E-46,4.09E-46,7.11E-46,1.06E-45  
+"#,XENON,Collaboration,2018.,arXiv-1805.12562,Phys.,Rev.,Lett.,121,111302,",,
```
Comment (kdund) [inline data/xenon1t_1ty.csv:1]: Adding commas at the end of a line messes up some .csv parsers-- best not to IMO

**Comment by kdund** [comment]
Hi, @yuema137 do you think we close this one since the ideas were used for the limit reformatting we did?

**Comment by yuema137** [comment]
@kdund I fully agree with closing this! Thanks for doing the formatting (I know it's a pain...)

**Pull request closed without merging**